packages/cloudi/cloudi.1.7.3/opam

```diff
@@ -0,0 +1,18 @@
+opam-version: "1.2"
+name: "cloudi"
+version: "1.7.3"
+maintainer: "Michael Truog <mjtruog@gmail.com>"
+authors: "Michael Truog <mjtruog@gmail.com>"
+homepage: "http://cloudi.org/"
+bug-reports: "https://github.com/CloudI/CloudI/issues"
```

Comment:
Isn't it more accurate to give the issue tracker for ```cloudi_api_ocaml``` instead ?

Reply:
@jpdeplaix All the source code is developed and tested in the `https://github.com/CloudI/CloudI/` repository, so I would prefer to have the issues there.  The separate `https://github.com/CloudI/cloudi_api_*` repositories are for development specific to creating programming language packages for the CloudI API.